addons/van_dml/src/README.md

```diff
@@ -0,0 +1,27 @@
+# van_dml.js: Add Auto-Append feature to VanJS
+
+van_dml.js adds a a new flavour of composition to VanJS. This is heavily inspired by the [DML](https://github.com/efpage/dml)-framework for dynamic page creation.
+
+Van_DML adds two main functions to VanJS.js. You can enable some new fuctions using
```

Comment:
Could you provide some instructions with examples on how Van_DML can be used (including what lines need to be added in the web page to import the right files)?